### PR TITLE
Document methods after definition.

### PR DESCRIPTION
**Pull request by MichaelHatherly**

### CODE REVIEW

File: test/replcompletions.jl [hunk 1]
```diff
@@ -161,7 +161,7 @@ c,r,res = test_complete(s)
 
 s = "\\a"
 c, r, res = test_complete(s)
-"\\alpha" in c
+("\\alpha" in c)
```
Comment (MichaelHatherly) [inline test/replcompletions.jl]: Some how this line was being treated as a docstring... not sure what's up there.

**Comment by MichaelHatherly** [comment]
cc @one-more-minute, @yuyichao.

**Comment by yuyichao** [comment]
I would still prefer the quoted syntax since it can just be treated as documenting a value. It doesn't conflict with this though so we could support both.

**Comment by MichaelHatherly** [comment]
I'm fine with either really. This way was simpler to add, no special casing as with the macro version.

**Comment by one-more-minute** [comment]
LGTM

File: test/replcompletions.jl (hunk 1 above)
Comment (tkelman) [inline test/replcompletions.jl]: worth a new issue?

Comment (MichaelHatherly) [inline test/replcompletions.jl]: Sure, was trying to reduce to something reproducible. I'll open one shortly.

**Comment by MichaelHatherly** [comment]
Rebased to remove the workaround needed in ``test/replcompletions.jl``, thanks to @yuyichao's fix in 77c6ffc9c3071546c70de734bca23cd2cd727189.

**Comment by one-more-minute** [comment]
Nice work as always

**Comment by MichaelHatherly** [comment]
Thanks

**Pull request merged**